src/main/resources/META-INF/webcontent/ecm_components.html

```diff
@@ -19,38 +19,69 @@
 
 <div data-eht-fragment="'content'" data-eht-render="'content'">
   <style>
-    #plugin_table .componentState_UNSATISFIED{
-      color: darkorange;
-    }
-    #plugin_table .componentState_FAILED{
-      color: red;
-    }
-  </style>
+#plugin_table .componentState_UNSATISFIED, .componentState_UNSATISFIED {
+  color: darkorange;
+}
+
+#plugin_table .componentState_FAILED, .componentState_FAILED {
+  color: red;
+}
+
+#plugin_table .componentState_STARTING, .componentState_STARTING {
+  color: purple;
+}
+
+#plugin_table .componentState_STOPPING, .componentState_STOPPING {
+  color: purple;
+}
+
+#plugin_table .componentState_INACTIVE, .componentState_INACTIVE {
+  color: darkgray;
+}
+</style>
   <p class="statline ui-state-highlight"
-    data-eht-text="'ECM component information: ' + ccMap.size() + ' container(s) in total.'">ECM component
-    information: 5 container(s) in total.</p>
+    data-eht-text="'ECM component information: ' + ccMap.size() + ' container(s) in total.'">ECM
+    component information: 5 container(s) in total.</p>
+  <p class="statline ui-state-highlight">
+    <span data-eht-text="'ACTIVE: ' + numberOfComponetntsByState[0]">ACTIVE: 0</span> <span
+      data-eht-text="'UNSATISFIED: ' + numberOfComponetntsByState[1]"
+      data-eht-attrprepend-class="numberOfComponetntsByState[1] > 0 ? 'componentState_UNSATISFIED' : ''">UNSATISFIED:
+      0</span> <span data-eht-text="'FAILED: ' + numberOfComponetntsByState[2]"
+      data-eht-attrprepend-class="numberOfComponetntsByState[2] > 0 ? 'componentState_FAILED' : ''">FAILED:
+      0</span> <span data-eht-text="'STARTING: ' + numberOfComponetntsByState[3]"
+      data-eht-attrprepend-class="numberOfComponetntsByState[3] > 0 ? 'componentState_STARTING' : ''">STARTING:
+      0</span> <span data-eht-text="'STOPPING: ' + numberOfComponetntsByState[4]"
+      data-eht-attrprepend-class="numberOfComponetntsByState[4] > 0 ? 'componentState_STOPPING' : ''">STOPPING:
+      0</span> <span data-eht-text="'INACTIVE: ' + numberOfComponetntsByState[5]"
+      data-eht-attrprepend-class="numberOfComponetntsByState[5] > 0 ? 'componentState_INACTIVE' : ''">INACTIVE:
+      0</span>
+  </p>
 
   <table id="plugin_table" class="tablesorter nicetable noauto ui-widget">
     <tbody data-eht-foreach="[({'ccSR', 'ccsrIdx'}) : ccMap.keySet()]" data-eht-render="'content'"
       data-eht-var="['cc' : ccMap.get(ccSR)]">
       <tr>
-        <th class="ui-widget-header" colspan="2"><a data-eht-var="['serviceId' : ccSR.getProperty('service.id')]"
-          data-eht-attr-href="appRoot + '/services/' + serviceId" data-eht-text="serviceId">1</a> - <span
-          data-eht-var="['mpp' : cc]" data-eht-text="mpp.getObjectClassDefinition(null, null).name">Component
-            name</span></th>
+        <th class="ui-widget-header" colspan="2"><a
```

Comment:
You probably used some auto-formatter that makes it impossible to see the real differences.